<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWinClass</class>
 <widget class="QMainWindow" name="MainWinClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>668</width>
          <height>291</height>
         </size>
        </property>
        <property name="autoScrollMargin">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Tempo rimanente:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Annulla</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shareBtn">
        <property name="text">
         <string>Condividi</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>Opzioni</string>
    </property>
    <widget class="QMenu" name="menuOnLine_2">
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Stato</string>
     </property>
     <addaction name="actionOnline"/>
     <addaction name="actionOffline"/>
    </widget>
    <widget class="QMenu" name="menuUsername_2">
     <property name="title">
      <string>Username</string>
     </property>
     <addaction name="actionNome"/>
     <addaction name="actionAnonimo"/>
    </widget>
    <widget class="QMenu" name="menuFoto_Profilo">
     <property name="title">
      <string>Foto Profilo</string>
     </property>
     <addaction name="actionAggiungi_foto_profilo"/>
     <addaction name="actionVolto_anonimo"/>
    </widget>
    <addaction name="menuOnLine_2"/>
    <addaction name="menuUsername_2"/>
    <addaction name="menuFoto_Profilo"/>
   </widget>
   <addaction name="menusettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOnline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Online</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionOffline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Offline</string>
   </property>
  </action>
  <action name="actionNome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </action>
  <action name="actionAnonimo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Anonimo</string>
   </property>
  </action>
  <action name="actionAggiungi_foto_profilo">
   <property name="text">
    <string>Aggiungi foto profilo</string>
   </property>
  </action>
  <action name="actionVolto_anonimo">
   <property name="text">
    <string>Volto anonimo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWin.qrc"/>
 </resources>
 <connections/>
</ui>
